core/src/main/scala/kafka/coordinator/transaction/TransactionCoordinator.scala

```diff
@@ -389,8 +393,8 @@ class TransactionCoordinator(brokerId: Int,
                       }
                     }
                   } else {
-                    info(s"Updating $transactionalId's transaction state to $newMetadata with coordinator epoch $coordinatorEpoch for $transactionalId failed since the transaction coordinator epoch " +
-                      s"has been changed to ${epochAndMetadata.coordinatorEpoch} after the transaction metadata has been successfully appended to the log")
+                    debug(s"The transaction coordinator epoch has been changed to ${epochAndMetadata.coordinatorEpoch} after $txnMarkerResult has " +
```

Comment:
nit: "has been changed" -> "has changed" and "has been successfully appended" -> "was successfully appended"